lib/src/base-styles/_nx-tables.scss

```diff
@@ -367,6 +367,41 @@
 
 .nx-cell__sort-btn {
   height: nx-table-variables.$table-cell-header-height;
+  position: relative;
+
+  > * {
+    position: relative;
```

Comment:
What's this for?

Reply:
> What's this for?

Without this, the stacking context was off. The `::before` element was covering up the button's `<span>` element containing the text.